abb_irb6700_support/urdf/irb6700_235_265_macro.xacro

```diff
@@ -0,0 +1,225 @@
+      <child link="${prefix}link_1" />
+      <axis xyz="0 0 1" />
+      <limit lower="-${radians(170)}" upper="${radians(170)}" effort="0" velocity="${radians(100)}" />
+    </joint>
+    <joint name="${prefix}joint_2" type="revolute">
+      <origin xyz="0.32 0 0.492" rpy="0 0 0" />
+      <parent link="${prefix}link_1" />
+      <child link="${prefix}link_2" />
+      <axis xyz="0 1 0" />
+      <limit lower="-${radians(65)}" upper="${radians(85)}" effort="0" velocity="${radians(90)}" />
+    </joint>
+    <joint name="${prefix}joint_3" type="revolute">
+      <origin xyz="0 0 1.135" rpy="0 0 0" />
+      <parent link="${prefix}link_2" />
+      <child link="${prefix}link_3" />
+      <axis xyz="0 1 0" />
+      <limit lower="-${radians(180)}" upper="${radians(70)}" effort="0" velocity="${radians(90)}" />
+    </joint>
+    <joint name="${prefix}joint_4" type="revolute">
+      <origin xyz="0 0 0.2" rpy="0 0 0" />
+      <parent link="${prefix}link_3" />
+      <child link="${prefix}link_4" />
+      <axis xyz="1 0 0" />
+      <limit lower="-${radians(300)}" upper="${radians(300)}" effort="0" velocity="${radians(170)}" />
+    </joint>
+    <joint name="${prefix}joint_5" type="revolute">
+      <origin xyz="1.1825 0 0" rpy="0 0 0" />
+      <parent link="${prefix}link_4" />
+      <child link="${prefix}link_5" />
+      <axis xyz="0 1 0" />
+      <limit lower="-${radians(130)}" upper="${radians(130)}" effort="0" velocity="${radians(120)}" />
+    </joint>
+    <joint name="${prefix}joint_6" type="revolute">
+      <origin xyz="0.165 0 0" rpy="0 0 0" />
+      <parent link="${prefix}link_5" />
+      <child link="${prefix}link_6" />
+      <axis xyz="1 0 0" />
+      <limit lower="-${radians(360)}" upper="${radians(360)}" effort="0" velocity="${radians(190)}" />
+    </joint>
+    <joint name="${prefix}tool_joint" type="fixed">
+      <origin xyz="0.035 0 0" rpy="0 ${radians(90)} 0" />
+      <parent link="${prefix}link_6" />
+      <child link="${prefix}tool0" />
+      <axis xyz="0 0 0" />
+    </joint>
+    <joint name="${prefix}cylinder_joint" type="revolute">
+      <origin xyz="-0.349 -0.194 0.35" rpy="0 0 0" />
+      <parent link="${prefix}link_1" />
+      <child link="${prefix}cylinder" />
+      <axis xyz="0 1 0" />
+      <mimic joint="${prefix}joint_2" multiplier="-0.25"/>
+      <limit lower="-3" upper="3" effort="0" velocity="100" />
+    </joint>
+    <joint name="${prefix}piston_joint" type="prismatic">
+      <origin xyz="0 0 0" rpy="0.20897 0 -${radians(90)}" />
+      <parent link="${prefix}cylinder" />
+      <child link="${prefix}piston" />
+      <axis xyz="0 1 0" />
+      <mimic joint="${prefix}joint_2" multiplier="0.15"/>
+      <limit lower="-3" upper="3" effort="0" velocity="100" />
+    </joint>-->
+
+    <!-- ROS base_link to ABB World Coordinates transform -->
+    <link name="${prefix}base" />
+    <joint name="${prefix}base_link_base" type="fixed">
+      <origin xyz="0 0 0.780" rpy="0 0 0"/>
```

Comment:
Is this correct? Other xacros in this repository (and over at `ros-industrial/abb`) have an all-zeros transform here typically.

Reply:
I think this was a mistake from working with other vendors' robots; I'll change it to match the other ABB models until I can verify on actual hardware